app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -67,7 +67,7 @@ def proxy_action_class
         end
 
         def required_proxy_selector_for(template)
-          if template.remote_execution_features.where(:label => 'ansible_run_capsule_upgrade').any?
+          if template.remote_execution_features.where(:label => FEATURES_THROUGH_INTERNAL_PROXY).any?
```

Comment:
This shouldn't be here now, right?

Reply:
Removed the entire method since it can use what we have in REX as-is